webpack/components/extensions/HostDetails/Tabs/DebsTab/DebInstallModal.scss

```diff
@@ -0,0 +1,3 @@
+#package-install-modal .pf-c-input-group input {
```

Comment:
```suggestion
#deb-install-modal .pf-c-input-group input {
```

If we don't want this CSS to also affect PackageInstallModal, maybe it's better to use a distinct ID.